companies/benjerry.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Ben & Jerry's
+slug: ben-&-jerrys
+blm_statements:
+  - url: https://www.benjerry.com/whats-new/2016/why-black-lives-matter
+    date_posted: 2016-10-06
```

Comment:
Awesome! I love that they were posting about this back in 2016.  Would you be able to add an additional record for their statement in June regarding George Floyd?